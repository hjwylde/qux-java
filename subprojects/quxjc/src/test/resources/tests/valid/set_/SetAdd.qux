package valid.set_

void main([str] args):
    print {1} + {0} + {'foo'}
    print {} + {3} + {}
    print {2} + {1} + {2}
    print {1, 1}

