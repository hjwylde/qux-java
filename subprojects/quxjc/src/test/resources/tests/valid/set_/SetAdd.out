{0, 1, foo}
{3}
{1, 2}
{1}
